.github/workflows/ci.yml

```
@@ -13,6 +13,29 @@ on:
   workflow_call: null
 
 jobs:
+  docs:
+    name: Build docs
+    runs-on: ubuntu-latest
+    timeout-minutes: 7
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - uses: actions/cache@v3
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('**/pyproject.toml', '**/*requirements.txt') }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r ./docs/requirements.txt
```

i think we need to make sure the local versions of the *staq packages are installed instead of the ones on pypi (might be a more efficient way to do this)
```suggestion
          pip install -e ./general-superstaq[dev] -e ./cirq-superstaq[dev] -e ./qiskit-superstaq[dev] -e ./supermarq-benchmarks[dev]
          pip install -r ./docs/requirements.txt
```

---

also might be worth updating docs/requirements.txt - it seems like a few of the requirements are out of date/redundant/conflict with requirements we use elsewhere. definitely double-check, but i think this is sufficient:
```
cirq-superstaq[dev]~=0.4.10
ipywidgets>=8.0.0
nbsphinx>=0.9.0
qiskit-superstaq[dev]~=0.4.10
sphinx-rtd-theme>=1.0.0
supermarq[dev]~=0.4.10
```